app/views/claims/pages/index.html.erb

```diff
@@ -1,28 +1,51 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">It works! 🎉</h1>
+<%= render(NavigationBar.new(
+    items: navigation_items(@provider),
+    current_path: request.path,
+    current_user: User.first,
```

Comment:
`current_user` is a helper method which can be used in the views and controllers.